src/server/stream_family_test.cc

```diff
@@ -236,6 +236,25 @@ TEST_F(StreamFamilyTest, Issue854) {
   EXPECT_THAT(resp, ErrArg("is not allowed"));
 }
 
+TEST_F(StreamFamilyTest, XGroupConsumer) {
+  Run({"xgroup", "create", "foo", "group", "$", "MKSTREAM"});
+  auto resp = Run({"xgroup", "createconsumer", "foo", "group", "bob"});
+  EXPECT_THAT(resp, IntArg(1));
+  Run({"xgroup", "createconsumer", "foo", "group", "alice"});
+  resp = Run({"xinfo", "groups", "foo"});
+  EXPECT_THAT(resp.GetVec()[3], IntArg(2));
+  Run({"xgroup", "delconsumer", "foo", "group", "alice"});
+  resp = Run({"xinfo", "groups", "foo"});
+  EXPECT_THAT(resp.GetVec()[3], IntArg(1));
+
+  resp = Run({"xgroup", "createconsumer", "foo", "group", "alice"});
+  EXPECT_THAT(resp, IntArg(1));
+
+  // ensure createconsumer doesn't create consumer that already exists
+  resp = Run({"xgroup", "createconsumer", "foo", "group", "alice"});
+  EXPECT_THAT(resp, IntArg(0));
+}
```

Comment:
Maybe expect one more error case?

Reply:
You mean adding one more error case to verify the commented behaviour right?

Reply:
Yes, I meant nogrouperror, but its not critical